Fix (Aave): Missing aTokens and order (#8182)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :bug:`-` Deposit events in Aave will now show in correct order with approval events.

Reviewer: ```suggestion
* :bug:`-` Deposit events in Aave will now show in the correct order.
```
